apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.js

```diff
@@ -1,30 +1,35 @@
-const ERR = require('async-stacktrace');
+// @ts-check
 const express = require('express');
 const asyncHandler = require('express-async-handler');
-const router = express.Router();
+const path = require('path');
+
+const { flash } = require('@prairielearn/flash');
+const sqldb = require('@prairielearn/postgres');
+const { logger } = require('@prairielearn/logger');
+const Sentry = require('@prairielearn/sentry');
+
 const opsbot = require('../../lib/opsbot');
 const github = require('../../lib/github');
-const { logger } = require('@prairielearn/logger');
```

Comment:
I don't necessarily mind moving these import lines but I just wanted to check that you are doing this deliberately, rather than having your VSCode/ESLint set up to do it automatically when you might not be intending to add extra noise to PRs?

Reply:
Yes, this was deliberate, though unnecessary - sorry for the noise! In this case specifically, I had to resolve a large conflict during a rebase and ended up just rewriting them from scratch in this grouped style. I'll get tooling in place soon so we don't ever have to think about this.